src/Symfony/Component/Form/Extension/Validator/ValidatorTypeGuesser.php

```diff
@@ -122,7 +122,13 @@ public function guessTypeForConstraint(Constraint $constraint)
 
             case 'Symfony\Component\Validator\Constraints\File':
             case 'Symfony\Component\Validator\Constraints\Image':
-                return new TypeGuess('Symfony\Component\Form\Extension\Core\Type\FileType', [], Guess::HIGH_CONFIDENCE);
+                return new TypeGuess(
```

Comment:
We always keep instructions on one line.

Reply:
Hello, i have updated this code to add condition before this instruction for define variable who contains the options.

```php
$options = [];
if ($constraint->mimeTypes) {
    $options = ['attr' => ['accept' => \is_array($constraint->mimeTypes) ? implode(',', $constraint->mimeTypes) : $constraint->mimeTypes]];
}

return new TypeGuess('Symfony\Component\Form\Extension\Core\Type\FileType', $options, Guess::HIGH_CONFIDENCE);
```

In this way, does the code seem clean to you ?

PS : I did not make a ternary condition because I is obligated to use a double ternary, and I find more complicated to reread